src/plugins/kibana/public/settings/sections/indices/add_data_steps/parse_csv_step/parse_csv_step.html

```diff
@@ -0,0 +1,57 @@
+<file-upload ng-if="!wizard.file" on-locate="wizard.file = file" upload-selector="button.upload">
+  <h2><em>Pick a CSV file to get started.</em>
+    Please follow the instructions below.
+  </h2>
+
+  <div class="upload-wizard-file-upload-container">
+    <div class="upload-instructions">Drop your file here</div>
+    <div class="upload-instructions-separator">or</div>
+    <button class="btn btn-primary btn-lg controls upload" ng-click>
+      Select File
+    </button>
+    <div>Maximum upload file size: 1 GB</div>
```

Comment:
Is this value available from config, etc so that it doesn't get out of sync with the actual limit?